index.js

```diff
@@ -34,16 +51,7 @@ function askForFoldersAccess(folder) {
 }
 
 module.exports = {
-  askForCalendarAccess: permissions.askForCalendarAccess,
-  askForContactsAccess: permissions.askForContactsAccess,
+  ...permissions,
```

Comment:
we probably also need to handle calls to `askForFoldersAccess` and `getAuthStatus` - `permissions.askForFoldersAccess.call(this, folder)` @ L33 for example probably blows up, does it not?

Reply:
These should also be covered in the additional unit test that runs on Linux 🙂 

https://github.com/codebytere/node-mac-permissions/blob/321a1f80896fa517983a79aaeffbd2d0af6e3f62/test/module.spec.js#L67-L68

The type/folder is validated, and the call is still to `permissions.askForFoldersAccess: () => undefined` from the original stub, so it behaves as expected with the test's assert. I originally had `nonMacResponse = (...args) => undefined` but then `...args` was just flagged as unused.

These two tests also run on all nodes to enforce consistency of the arg cross-platform.
https://github.com/codebytere/node-mac-permissions/blob/321a1f80896fa517983a79aaeffbd2d0af6e3f62/test/module.spec.js#L10-L14
https://github.com/codebytere/node-mac-permissions/blob/321a1f80896fa517983a79aaeffbd2d0af6e3f62/test/module.spec.js#L40-L44
The only notable difference is that the `index.d.ts` does not specify the new `undefined` behavior.  I felt that was acceptable as this is explicitly a mac-only module, this enhancement is mainly to allow type-safety with a standard import in local projects.